python/pyspark/sql/context.py

```diff
@@ -21,6 +21,7 @@
 
 if sys.version >= '3':
     basestring = unicode = str
+    long = int
```

Comment:
actually @adrian-wang I think Python's int is larger than 32 bit, so we don't need this long thing.